sdk/tests/AccountIdTest.cc

```diff
@@ -91,6 +102,7 @@ TEST_F(AccountIdTest, ConstructFromString)
   EXPECT_THROW(AccountId("abc"), std::invalid_argument);
   EXPECT_THROW(AccountId("o.o.e"), std::invalid_argument);
   EXPECT_THROW(AccountId("0.0.1!"), std::invalid_argument);
+  EXPECT_THROW(AccountId(testNumTooBigStr + '.' + testNumTooBigStr + '.' + testNumTooBigStr), std::invalid_argument);
```

Comment:
Since the error checking is split into different blocks in the original code, I would add an `EXPECT_THROW` for each individual string being too big. Currently, you'll hit the same error block each time and early out

Reply:
Fixed.